### PR TITLE
Added testcase for distributedMap to checkpoint fat suite

### PR DESCRIPTION
**Pull request by KeenanMcCarthy**
Testing distributedMap cache timeout with checkpoint and restore

### CODE REVIEW

**Comment by KeenanMcCarthy** [comment]
#build

**Comment by KeenanMcCarthy** [comment]
#build

**Comment by KeenanMcCarthy** [comment]
#build
#libby

File: dev/io.openliberty.checkpoint_fat/.classpath [hunk 1]
```diff
@@ -32,11 +32,6 @@
 	<classpathentry kind="src" path="test-applications/virtualhost/src"/>
 	<classpathentry kind="src" path="test-applications/DefaultPrincipalMappingApp/src"/>
 	<classpathentry kind="src" path="test-applications/concurrentApp/src"/>
-	<classpathentry kind="con" path="aQute.bnd.classpath.container"/>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8">
-		<attributes>
-			<attribute name="module" value="true"/>
-		</attributes>
-	</classpathentry>
```
Comment (tjwatson) [inline dev/io.openliberty.checkpoint_fat/.classpath]: I'm not sure we should remove these.

File: dev/io.openliberty.checkpoint_fat/publish/servers/checkpointMapCache/bootstrap.properties [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+bootstrap.include=../testports.properties
+com.ibm.ws.logging.trace.specification=*=info:checkpoint=all
+io.openliberty.checkpoint.dump.threads=true
```
Comment (tjwatson) [inline dev/io.openliberty.checkpoint_fat/publish/servers/checkpointMapCache/bootstrap.properties]: `websphere.java.security.exempt=true` needs to be added to bootstrap.properties

File: dev/io.openliberty.checkpoint_fat/.classpath (hunk 1 above)
Comment (KeenanMcCarthy) [reply dev/io.openliberty.checkpoint_fat/.classpath]: Agreed, resolving

File: dev/io.openliberty.checkpoint_fat/publish/servers/checkpointMapCache/bootstrap.properties (hunk 2 above)
Comment (KeenanMcCarthy) [reply dev/io.openliberty.checkpoint_fat/publish/servers/checkpointMapCache/bootstrap.properties]: Added

**Comment by KeenanMcCarthy** [comment]
#build

**Review by tjwatson — APPROVED** [review]

**Comment by tjwatson** [comment]
#libby

**Comment by tjwatson** [comment]
Fixes #21324

**Pull request merged**